cmd/noms/splore/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["stage-3", "react"]
```

Comment:
I spent some time trying to make this less specialised, as in hours, but I couldn't make the combination of flow/eslint/webpack work, so I gave up. I.e. webpack kept crashing and/or eslint - I can't remember (besides which: given this calls `open` there is a fair chance it will open Safari - does that have the features too?).